src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Classnames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+enum Response {
+  title = 'Title',
+  error = 'Error',
+}
+
+type Props = {
+  addMovie: (movie:Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [search, setSearch] = useState<string>('');
+  const [answear, setAnswear] = useState<Movie | {} | ResponseError>({});
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [error, setError] = useState<boolean>(false);
+
+  const onChangeHandel = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearch(event.target.value);
+    if (error) {
+      setError(false);
+    }
+  };
+
+  const requstByMovie = async () => {
+    setIsLoading(true);
+    const response = await getMovie(search);
+
+    setIsLoading(false);
+
+    if (Response.title in response) {
+      setAnswear({
+        title: response.Title,
+        description: response.Plot,
+        imgUrl: response.Poster !== 'N/A'
+          ? response.Poster
+          : 'https://via.placeholder.com/360x270.png?text=no%20preview',
+        imdbUrl: `https://www.imdb.com/title/${response.imdbID}`,
+        imdbId: response.imdbID,
+      });
+    }
+
+    if (Response.error in response) {
+      setAnswear(response);
+      setError(true);
+    }
+  };
+
+  const addAndClear = (movie: Movie) => {
+    addMovie(movie);
+    setSearch('');
+    setAnswear({});
+  };
+
   return (
     <>
-      <form className="find-movie">
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+          requstByMovie();
+        }}
```

Comment:
create a separate fucntion for this
```suggestion
        onSubmit={handleSubmit}
```